docs/basics/Epics.md

```diff
@@ -220,21 +224,16 @@ Finally, redux-observable provides a utility called [`combineEpics()`](../api/co
 ```js
 import { combineEpics } from 'redux-observable';
 
-const rootEpic = combineEpics(
-  pingEpic,
-  fetchUserEpic
-);
+const rootEpic = combineEpics(pingEpic, fetchUserEpic);
 ```
 
 Note that this is equivalent to:
 
 ```js
 import { merge } from 'rxjs/observable/merge';
 
-const rootEpic = (action$, store) => merge(
-  pingEpic(action$, store),
-  fetchUserEpic(action$, store)
-);
+const rootEpic = (action$, store) =>
+  merge(pingEpic(action$, store), fetchUserEpic(action$, store));
```

Comment:
While I don't mind the new variant, I imagine the prior version is easier for newcomers to grok. Not a big deal, curious what others think.

Reply:
The old version is prettier and easier to understand but the hassle "prettier" saves is worth accepting a few weird choices for.